doc/nrf/app_dev/ap_protect/index.rst

```diff
@@ -26,16 +26,16 @@ For detailed information, refer to the hardware documentation.
 .. list-table:: AP-Protect implementations
    :header-rows: 1
 
-   * - Implementation type
+   * - AP-Protect implementation type
      - Default factory state
      - How to enable
      - How to disable
-   * - AP-Protect controlled by hardware
+   * - Hardware
      - Disabled
      - Writing ``UICR.APPROTECT`` to ``Enabled`` and performing reset.
      - Issuing an ``ERASEALL`` command via CTRL-AP.
        This command erases the flash, UICR, and RAM, including ``UICR.APPROTECT``.
-   * - AP-Protect controlled by hardware and software
+   * - Hardware and software
      - Enabled
      - After it is disabled, a reset or a wake enables the access port protection again.
      - Issuing an ``ERASEALL`` command via CTRL-AP.
```

Comment:
Doesn't this erase `UICR.APPROTECT` being programmed to `HwDisabled`, and thus it never stays disabled?

Or am I not understandinf the table correctly?

Reply:
You erase, then write `UICR.APPROTECT` to `HwDisabled`, program, and avoid using a reset or a wake until you're done working and want to re-enable AP-Protect.